proto/storage/compliance_operator_v2.proto

```diff
@@ -135,6 +135,24 @@ message ComplianceOperatorClusterScanConfigStatus {
   string cluster_name = 5;
 }
 
+// Next Tag: 7
+message ComplianceOperatorBenchmarkV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
```

Comment:
I think we also need the Publish Date here. Created a ticket for that, doesn't need to be done in this PR: https://issues.redhat.com/browse/ROX-24395

Reply:
Perfect 👍